{
  Main main=new Main(args);
  main.setBuildCompleter(new Main.BuildCompleter(){
    public void exit(    Throwable failure){
      throw new BuildCompletedError(failure);
    }
  }
);
  try {
    main.execute();
    fail();
  }
 catch (  BuildCompletedError e) {
    assertThat(e.getCause(),notNullValue());
    throw e.getCause();
  }
}
